franklin/builder/models.py

```diff
@@ -8,16 +8,16 @@ class Site(models.Model):
     """ Represents a 'deployed' or soon-to-be deployed static site.
 
     :param id: A unique site identified
+    :param repo_name: The full repo name of project on github
     :param git_hash: The git hash of the deployed code
-    :param repo_name: The repo name of the site to check for 're-deploys'
     :param path: The path of the site on the static server
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    git_hash = models.CharField(max_length=50)
-    repo_name = models.CharField(max_length=20, default='')
+    repo_name = models.CharField(max_length=100)
```

Comment:
Actually, repo_name is effectively a unique ID from the row in the DB. So it can't be blank ever. I actually set it as `unique=True`, but then I got an odd migration error locally. In my next PR, I want to switch this to be unique and drop the `id` attribute as it's not needed. That's another thing that threw me an odd error when I tried. Skipped as I wanted to get an MVP out there.